### DISCUSSION TITLE
Non-modal pop-up windows that can be moved?

### DISCUSSION THREAD

**Discussion by HubKing** [Megabit/Blazorise#4376] [Q&A] [answered]
Does Blazorise have non-modal virtual pop-up windows to display a Blazor component? By "virtual", I mean that they are not a window in the OS, nor a real tab in the browser, but are just a DIV with higher z-order.

I need virtual pop-up windows that can stay above the main page, allow users to interact with the page, can be moved, and have multiple pop-ups opened at the same time. Something like this https://www.cssscript.com/demo/draggable-popup-window-engine/

![image](https://user-images.githubusercontent.com/10900489/206529585-b1958397-8a8d-4a15-b093-ede3f61dcdc6.png)

**Comment by stsrki** [comment, marked as answer]
Hey @HubKing . There's nothing like that in Blazorise. It seems like a quite niche feature, so I'm not sure we would make it any time soon.

**Reply by HubKing** [reply]
Well, there already seemed to be "Modal" windows, so I thought there might be non-modal windows. That is fine, if it does not exist, because I could use actual pop-ups, but since Blazor does not seem to like opening any new windows, I thought maybe I should use virtual layers instead. As for usage, I was trying to open the details of items in a list. I could open it in a modal pop-up, but then again, why limit that to only one and block the whole UI? The user might want to open the details of two items and compare them side-by-side.
